distribution/src/main/resources/bin/plugin

```diff
@@ -23,7 +23,7 @@ ES_HOME=`cd "$ES_HOME"; pwd`
 
 # Sets the default values for elasticsearch variables used in this script
 if [ -z "$CONF_DIR" ]; then
-  CONF_DIR="${packaging.plugin.default.config.dir}"
+  CONF_DIR="$ES_HOME/config"
```

Comment:
This should not be changed, the value `${packaging.plugin.default.config.dir}` is replaced at build time this line allows us to define the default value when `CONF_DIR` is not explicitly defined in the current env.